docroot/themes/humsci/humsci_basic/scripts/no-js.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Add .js class, and remove .no-js (if JavaScript loads on the page)
+ * This allows us to apply specific styles for no-JS solutions.
+ *
+ * We load this as a standalone script so we can prioritize it to the
+ * head of the document and avoid a flash of unstyled content. See:
+ * https://stackoverflow.com/a/12410668/1154642
+ */
+document.documentElement.classList.add('js');
+document.documentElement.classList.remove('no-js');
```

Comment:
drupal core `modernizer`  library does this already. this is redundant

Reply:
Modernizer isn't currently being loaded in our themes, mostly because we haven't needed it thus far. However, we did want to support this one feature (js/no-js classes), so that's what this file was intended to do.

I suppose we could load core's modernizer instead of this custom implementation if you think that's more appropriate. Want us to do that instead?

(Modernizer does come with some extra features we wouldn't be using, but the file's only 4.5k, so not that big of a deal).

Reply:
yeah adding `core/modernizer` would be a preferred approach